### ISSUE TITLE
Jagex Launcher not found

### ISSUE THREAD

**Issue by lvf23** [TormStorm/jagex-launcher-linux#104]
**Describe the bug**

Installation goes well and exits with 0 status, but when it launches Jagex Launcher, it appears a dialog with the message: 

>"The file /home/lucas/Games/jagex-launcher/drive_c/Program Files (x86)/Jagex Launcher/JagexLauncher.exe can not be found" (Transcribed from pt-br).

Checking the path "/home/lucas/Games/jagex-launcher/drive_c/Program Files (x86)/Jagex Launcher/". The executable is not there. Only appear the folder "Games" with two other folders inside it called 'HDOS' and 'RuneLite'. I don't plan to play either of these versions of RuneScape. 

**To Reproduce**

Followed the installation normally as described in the documentation of this repository. 

1. Installed the Flatpak via apt.
2. Installed Lutris via .deb delivered in its GitHub repository.
3. Downloaded from master 'jagexlauncher.yml'.
4. Opened Lutris, clicked in plus button (+), then 'install from a script'.
5. Select 'jagexlauncher.yml', then, appear an option "Wine Installer" with the button 'Install', clicked in 'Install'.
6. Appeared the option where I would like to install: "/home/lucas/Games/jagex-launcher", then, I left it as it is.
7. Appeared a screen with sources of the download of dependencies, I left it as it is. Clicked on 'Install'.
8. Then it appears a dialog of Lutris with some log installing, additional dialogs of Mono and Gecko appear too, then they close automatically after installing these dependencies.
9. The dialog of Lutris freezes and appears a dialog saying that it freezes, but that's normal second documentation, then, I let the installation continue until the end.
10. Right after the dialog of Lutris closes, it tries to launch the Jagex Launcher, triggering the dialog as described before.

**Expected behavior**

It was expected to run the Jagex Launcher after its installation. 

**Screenshots**

![jagexlaunchernotfound](https://github.com/TormStorm/jagex-launcher-linux/assets/146987511/7aff69fc-0b15-40f4-87f1-92a1a56a89fd)

**Hardware information**
In Lutris click the hamburger menu and select preferences, hardware information and copy this.

```
[System]
OS:              Ubuntu 22.04 Jammy Jellyfish
Arch:            x86_64
Kernel:          6.2.0-34-generic
Desktop:         XFCE
Display Server:  x11

[CPU]
Vendor:          GenuineIntel
Model:           Intel(R) Core(TM) i5-7200U CPU @ 2.50GHz
Physical cores:  2
Logical cores:   4

[Memory]
RAM:             7.6 GB
Swap:            0.0 GB

[Graphics]
Vendor:          Intel
OpenGL Renderer: Mesa Intel(R) HD Graphics 620 (KBL GT2)
OpenGL Version:  4.6 (Compatibility Profile) Mesa 23.0.4-0ubuntu1~22.04.1
OpenGL Core:     4.6 (Core Profile) Mesa 23.0.4-0ubuntu1~22.04.1
OpenGL ES:       OpenGL ES 3.2 Mesa 23.0.4-0ubuntu1~22.04.1
Vulkan Version:  1.3.204
Vulkan Drivers:  Intel(R) HD Graphics 620 (KBL GT2) (1.3.238), NVIDIA GeForce 940MX (1.3.242)
```

**Additional context**

* I would like to play RS3. I didn't reach the step to install Runescape Client via Flatpak since I was stuck in that Jagex Launcher error step.

* Jagex Launcher apparently is installed in Lutris since appear the icon of Jagex Launcher that I can select and press play, but it triggers the initial dialog that appears the error described.

**Comment by lvf23** [comment]
Looks like it is linked with https://github.com/TormStorm/jagex-launcher-linux/issues/96. I did the steps of that issue by clicking with the Jagex Launcher Icon in Lutris and right clicked selecting ''Install another version" Then, then appeared the dialog described in the steps:

>7.Appeared a screen with sources of the download of dependencies, I left it as it is. Clicked on 'Install'.

Here is a screenshot of this screen:

![sourcesscreen](https://github.com/TormStorm/jagex-launcher-linux/assets/146987511/be8312b4-4965-4e0e-9ef2-d7fa156a3a04)

After following the steps of that screen again:

1.  it appeared the installer of Jagex Launcher (Windows style) and then I removed the content of the installation folder manually  (since Jagex Launcher requires an empty folder to install it).
2. Following that installer "next buttons", it installed Jagex Launcher successfully.
3. Afterward, appeared the login screen of Launcher to insert the e-mail and password.
4. Then, I installed the RuneScape Client using it in the terminal as described in the wiki: 
```flatpak install flathub com.jagex.RuneScape```
5. Clicked in Install in Jagex Launcher, then it started the RuneScape Client successfully.
6. A final consideration is that It would be useful to fix this issue or add that error in the FAQ.

**Note:** After the installation of Launcher, it will appear two Jagex Launcher icons in Lutris, you can safely delete the icon that appears semitransparent.

**Comment by nmlynch94** [comment]
@lvf23 Hello, thanks for the thorough steps given. There isn't much difference between those two scripts you chose, so I'm not sure why the second one worked. There are likely errors in the installation logs that we'd need to see to diagnose the issue. 

There is definitely something wrong in the error handling of the script, though. It shouldn't finish the installation if it fails to properly download the jagex launcher.

**Comment by nmlynch94** [comment]
@lvf23 Thank you. This has been a recurring issue "Timeout waiting for windows to load". Seems like it's a race condition that happens in the installer due to the underlying installer technology used. Have been unable to root cause it at this point, I suspect it's related to gecko somehow but am not sure.

**Comment by TormStorm** [comment]
This should be addressed with version 1.0.0
